fidesctl/demo_resources/demo_system.yml

```diff
@@ -24,7 +24,32 @@ system:
         data_categories:
           #- user.provided.identifiable.contact # uncomment to add this category to the system
           - user.derived.identifiable.device.cookie_id
-        data_use: advertising
+        data_use: third_party_sharing.personalized_advertising.marketing_co
         data_subjects:
           - customer
         data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+
+  - fides_key: demo_payroll_system
+    name: Demo Payroll System
+    description: Handle all required payroll data for the company
+    system_type: Service # is this documented somewhere currently?
+    privacy_declarations:
+      - name: Handle payroll activities
+        data_categories:
+          - user.provided.identifiable.financial
+        data_use: third_party_sharing.legal_obligation.hmrc
+        data_subjects:
+          - employee
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+      - name: Handle tax compliane with Her Majoesty's Revenue and Customs (HMRC)
+        data_categories:
+          - user.provided.identifiable.financial
+        data_use: third_party_sharing.legal_obligation
+        data_subjects:
+          - employee
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+        data_recipients:
```

Comment:
@iamkelllly - sorry about the delay getting an example out for this! Would you mind sharing if this is what you were thinking as far as an additional taxonomy for a data recipient (and possible other related attributes?)

Reply:
Looking at this example, it seems odd that the `purpose_of_processing` would *belong* to the `data_recipient` like this, e.g. "The recipient of the data is HMRC, and the purpose of processing for this recipient is a legal obligation".

My mind naturally wants to invert this, like "The purpose of processing for this data is a legal obligation, and the recipient is HMRC".

Reply:
I can definitely see it being flipped on it's head, good call 👍🏽 after our recent discussions it seemed clear that they should be coupled in some way is all.

As an example, a dataset containing employee info could provide personnel information for both performance management/salary reviews as well as for reporting to a tax authority. If flipped to have the `purpose_of_processing` as the higher level you could potentially have a list of recipients that have access for the same reason (e.g. HMRC, IRS, state tax authorities, etc.) instead of duplicating it for each recipient.